Tests/RoslynPluginGeneratorTests/NuGetPackageHandlerTests.cs

```diff
@@ -17,22 +19,211 @@ public class NuGetPackageHandlerTests
     {
         public TestContext TestContext { get; set; }
 
+        private const string testPackageName = "testPackage";
+        private const string dependentPackageName = "dependentPackage";
+
+        private const string releaseVersion = "1.0.0";
+        private const string preReleaseVersion = "1.0.0-RC1";
+
         [TestMethod]
-        public void NuGet_TestDependencyResolutionFailure()
+        public void NuGet_TestPackageDownload_Release_Release()
         {
             // Arrange
             string testDir = TestUtils.CreateTestDirectory(this.TestContext);
+            string testDownloadDir = Path.Combine(testDir, "download");
```

Comment:
+1
It should be clear when reading a test which bits of data are important (the ids, version and dependency relationship) and which aren't (authors, description). At the moment the intent is lost in the noise.